sdk/ml/azure-ai-ml/azure/ai/ml/_arm_deployments/arm_templates/workspace_base.json

```diff
@@ -128,6 +128,20 @@
                 "description": "Determines whether or not new ApplicationInsights should be provisioned."
             }
         },
+        "logAnalyticsName": {
```

Comment:
why do we need both logAnalyticsName and LogAnalyticsArmId?

Reply:
log analytics name is used for naming the resource to be created while log analytics arm id is used in the creation of the app insights resource